internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,61 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	separateData := strings.Split(data, ",")
+	if len(separateData) != 2 {
+		log.Println("Ошибка парсинга")
+		return 0, 0, fmt.Errorf("ошибка парсинга")
+	}
+	steps, err := strconv.Atoi(separateData[0])
+	if err != nil {
+		log.Println("Ошибка приведения количества шагов в целое число")
+		return 0, 0, fmt.Errorf("ошибка приведения количества шагов в целое число")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```